.github/workflows/cno-ci.yml

```diff
@@ -0,0 +1,131 @@
+name: ovn-cno-ci
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+
+env:
+  GO_VERSION: 1.13.4
+  K8S_VERSION: v1.18.2
+  KIND_CLUSTER_NAME: ovn
+  BUILD_CNO: true
+  BUILD_OVN: true
+
+jobs:
+  k8s:
+    name: Build k8s
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set up environment
+        run: |
+          export GOPATH=$(go env GOPATH)
+          echo "::set-env name=GOPATH::$GOPATH"
+          export PATH=$GOPATH/bin:$PATH
+          echo "::add-path::$GOPATH/bin"
+
+      - name: Cache Kubernetes
+        id: cache-k8s
+        if: github.event_name == 'push' || github.event_name == 'pull_request'
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.GOPATH }}/src/k8s.io/kubernetes/
+          key: k8s-go-2-${{ env.K8S_VERSION }}
+
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ env.GO_VERSION }}
+        id: go
+        if: steps.cache-k8s.outputs.cache-hit != 'true'
+
+      - name: Build and install Kubernetes
+        if: steps.cache-k8s.outputs.cache-hit != 'true'
+        run: |
+          set -x
+          rm -rf $GOPATH/src/k8s.io/kubernetes
+          git clone --single-branch --branch $K8S_VERSION https://github.com/kubernetes/kubernetes.git $GOPATH/src/k8s.io/kubernetes/
+          pushd $GOPATH/src/k8s.io/kubernetes/
+          make WHAT="test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo cmd/kubectl"
+          rm -rf .git
+
+  e2e:
+    name: e2e
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - shard-n
+          - shard-np
+          - shard-s
+          - shard-other
+        ha:
+          - enabled: "false"
+            name: "noHA"
+    needs: k8s
+    env:
+      JOB_NAME: "${{ matrix.target }}-${{ matrix.ha.name }}"
+      KIND_HA: "${{ matrix.ha.enabled }}"
```

Comment:
s/KIND_HA/OVN_HA/

Reply:
Thanks! Will be re-syncing this with upstream, I'm currently trying to get KIND to build, going to try bumping versions.

Reply:
just ping me in slack if you are blocked, happy to help debugging this